docs/guide/en/built-in-rules-callback.md

```diff
@@ -391,5 +391,16 @@ $rules = [
 $result = (new Validator())->validate($data, $rules);
 ```
 
+[`callables`]: https://www.php.net/manual/en/language.types.callable.php
+[custom rule]: creating-custom-rules.md
 [YAML]: https://en.wikipedia.org/wiki/YAML
-[yaml_parse docs]: https://www.php.net/manual/en/function.yaml-parse.php
+[PHP `yaml`]: https://www.php.net/manual/en/book.yaml.php
+[`yaml_parse()`]: https://www.php.net/manual/en/function.yaml-parse.php
+[`DTO`]: https://pt.wikipedia.org/wiki/Data_transfer_object
+[`when`]: conditional-validation.md#when
+[`public`]: https://www.php.net/manual/en/language.oop5.visibility.php
+[`protected`]: https://www.php.net/manual/en/language.oop5.visibility.php
+[`private`]: https://www.php.net/manual/en/language.oop5.visibility.php
+[`static`]: https://www.php.net/manual/en/language.oop5.static.php
+[`method`]: built-in-rules-callback.md#for-property
```

Comment:
Why need extract all links to footer? It makes sense if the links are repeated.

Reply:
During translation, I found links in the footer that were not duplicates. But also, mainly, I found it convenient for all the links to be in the footer, perhaps because it is more organized, perhaps because it makes future maintenance easier.
I was inspired by whoever did the validator documentation. Whoever it was, I thought it was worth following their example.

Reply:
I think is too much links. Not need all keywords make as link.
May be keep english docs as is and add translation only?

Reply:
What about it?

Please, don't mark non-resolved conversations as resolved.

Reply:
Sorry. I had removed the additional links mentioned, clicked ok and marked it as resolved. But I didn't explain....